### PR TITLE
Add missing props to Navbar.Collapse

### PR DESCRIPTION
**Pull request by thejohnfreeman**
I tried testing this change in a dependent project, but installing the package from local clone just results in `Cannot find module 'react-bootstrap/Navbar'`. To complicate matters, the build setup seems closely tied to the contributors' development environment. The second commit in this PR fixes the build from a clean state; it only ever worked from a dirty state. There is no `prepare` NPM script: is that intentional?

At the least, this can serve as a starting point.

/cc @jquense

### CODE REVIEW

**Comment by thejohnfreeman** [comment]
I was able to release [my fork of the library](https://www.npmjs.com/package/@thejohnfreeman/react-bootstrap) today for testing, and it seems to work.

File: tools/build.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const libRoot = path.join(__dirname, '../lib');
 const distRoot = path.join(libRoot, 'dist');
 const esRoot = path.join(libRoot, 'es');
 
-const clean = () => fse.existsSync(libRoot) && fse.remove(libRoot);
+const clean = async () => fse.existsSync(libRoot) && fse.remove(libRoot);
```
Comment (taion) [inline tools/build.js]: just don't await this below; this isn't `async`

Comment (jquense) [reply tools/build.js]: `remove` is

Comment (taion) [reply tools/build.js]: ah shoot, we didn't merge https://github.com/react-bootstrap/react-bootstrap/pull/3501

Comment (taion) [reply tools/build.js]: okay, can undo changes here now that we've merged

Comment (thejohnfreeman) [reply tools/build.js]: Done.

**Pull request merged**